client/src/components/routes/event/EventDetailSlide.js

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import styled from "styled-components";
+
+import AppButton from "components/common/AppButton";
+import AppCardImage from "components/common/AppCardImage";
+import AppHeading from "components/common/AppHeading";
+import AppSubheading from "components/common/AppSubheading";
+import AppText from "components/common/AppText";
+import { media } from "style/style-utils";
+
+const SlideArticle = styled.article`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+
+  ${media.mdScreen`
+    flex-direction: row;
+    align-items: center;
+    height: 100%;
+  `}
+`;
+
+const SlidePoster = styled(AppCardImage)`
+  width: 35%;
+`;
+
+const TextBox = styled.div`
+  width: 60%;
+  height: 80%;
+`;
+
+const SlideHeadingH2 = AppHeading.withComponent("h2");
+const SlideSubheadingP = AppSubheading.withComponent("p");
+const SlideTextP = AppText.withComponent("p");
+
+const SlideDescription = AppText.withComponent("p").extend`
+  width: 75%;
+  margin-bottom: 2rem;
+`;
+
+const formatLogistics = (formattedAllDates, formattedAllTimes, allVenues) => {
```

Comment:
You can write this directly in the component with a [`.map`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Map).

(As an aside, prefer array methods over pushing to an already-made array since the latter is a form of mutation, which is [bad practice](https://alistapart.com/article/why-mutation-can-be-scary).)